src/slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[slice(1, 2, 1)])
```

Comment:
This works, but the problems were asking you to use python slice notation, not the list slice method. Here's some examples https://www.pythoncentral.io/how-to-slice-listsarrays-and-tuples-in-python/